meson.build

```diff
@@ -223,8 +233,6 @@ else
 		]
 		moc_h += [
 			'src/dblsqd/feed.h',
-			'src/dblsqd/release.h',
-			'src/dblsqd/semver.h',
```

Comment:
Don't we need to include these header files?

Reply:
`moc_h` is only for header files that include classes which derive from `QObject`. It tells the Qt tooling to inject a bunch of other things into the source code before compilation. Neither of these classes derive from `QObject` so it's unnecessary to moc them. In fact, meson currently gives warnings because they are included and there is nothing for it to do.

Reply:
You can tell if moc'ing is necessary by looking for a `Q_OBJECT` macro inside the source; this is what the tool looks for. i.e. for `src/dblsqd/feed.h`

```
class Feed : public QObject
{
    Q_OBJECT
```